ci/component.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15
```

Comment:
Not sure if this should be a pinned version like build and unit. I would probably keep it the same as them though. Open to challenge.